### ISSUE TITLE
Very parse, no json. Sad shibe.

### ISSUE THREAD

**Issue by Akii** [dogescript/DSON.djs#4]
Input DSON: such "foo" is "bar" wow
Output JSON: { foo: 'bar' }

but should be: {"foo": "bar"}

No parse :(

PS: Nice job though :)

**Comment by remixz** [comment]
Ah, yeah, I suppose that's true. I modeled `DSON.parse` after `JSON.parse`, which parses JSON into a JS object. The best solution might be to `JSON.stringify` the object that `DSON.parse` returns.

**Comment by Akii** [comment]
Indeed, that seems to fix it. Is that something DSON.parse should do?

**Comment by Akii** [comment]
Speaking of parsing, I've found another "issue" it seems.

Parsing this https://github.com/Akii/Fleapack.Shiba.Doge/blob/master/Configuration/Settings.dson with DSON.djs results in following JSON (after putting it through JSON.stringify):

```
{
  "Fleapack": {
    "Shiba": {
      "Doge": {
        "dogeifyModes": "0.9"
      }
    }
  }
}
```

According to the specification the modes should be an array.. and so on.
